src/picknik_ur_multi_arm_config/config/moveit/multi_ur_servo.yaml

```diff
@@ -35,7 +35,7 @@ smoothing_filter_plugin_name: "online_signal_smoothing::ButterworthFilterPlugin"
 low_pass_filter_coeff: 1.5  # Larger --> trust the filtered data more, trust the measurements less.
 
 ## MoveIt properties
-move_group_name: multi_arm_manipulator  # Often 'manipulator' or 'arm'
+move_group_name: first_manipulator  # Often 'manipulator' or 'arm'
```

Comment:
I'm not certain why this has to be first_manipulator to work

Reply:
I think servo can only control a chain of links with a single tip frame. That is not the case for bigger groups. But we can enable group switching next sprint